DigitalOffice/Services/FileService/Mappers/FileMapper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FileService.Database.Entities;
+using FileService.Mappers.Interfaces;
+using FileService.Models;
+using System;
+
+namespace FileService.Mappers
+{
+    public class FileMapper : IMapper<DbFile, FileRequest>, IMapper<FileRequest, DbFile>
```

Comment:
maybe we should do two different mappers here (Solid)

Reply:
Agree. Will fix

Reply:
@DenisKrivoruchko11 @DarkFelas Spartak advised that it can be implemented via a single mapper.

Come on, think about it. We don't create a whole new repository just for adding files and then another one for editing files.

Solid is actually in play here: S -- Mapper Maps, I - Inherites needed interfaces.

The extra classes are junk and unnecessary I think.

Reply:
but we have two methods with name `Map` and I think, it is not good

Reply:
Yeah, I mentioned above that it's SOLID -- actually, 50/50.
So either way SOLID is broken this time, no matter what we do -- clutter the interfaces or follow Single Responsibility and create extra files.

Reply:
I think it's ok